### ISSUE TITLE
[Experiments - Authenticators] Graceful erroring if no keys found.

### ISSUE THREAD

**Issue by My1** [keys-pub/app#8]
might be better to just say that no devices found or whatever instead of cryptic error numbers
![image](https://user-images.githubusercontent.com/6696524/80925922-fc404200-8d93-11ea-86d5-80b29a6822f9.png)

**Comment by gabriel** [comment]
Yeah, it is still experimental, I haven't quite got all the edge cases yet.

What environment are you running on again? Assuming you don't have any devices plugged in?

**Comment by My1** [comment]
> What environment are you running on again?
Kubuntu 18.04

> Assuming you don't have any devices plugged in?
yes I didnt have at start when I got the error.
